src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,16 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import classnames from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <>
+    <article className={classnames('TodoInfo', {
+      'TodoInfo--completed': todo.completed === true,
```

Comment:
if `completed` field is boolean you can simplify it
```suggestion
      'TodoInfo--completed': todo.completed,
```